src/pageComponents/home.js

```diff
@@ -0,0 +1,111 @@
+import { topRatedMovieUrl, topRatedTVShowsUrl, genreUrl } from "../movies/constants/Constants";
+import React, { useEffect, useState } from "react";
+import TopRatedMovie from "../movies/api/GetTopRated";
+import ActionsButtons from "../genericComponents/Carousel";
+
+export default function Home() {
+  const [respTopRatedMovie, setRespTopRatedMovie] = useState([]);
+  const [respTopRatedTVShows, setResprespTopRatedTVShows] = useState([]);
+
+  //top rated movie
+  useEffect(() => {
+    const resultTopRatedMovie = TopRatedMovie(topRatedMovieUrl);
+    const resultMovieGenre = TopRatedMovie(genreUrl);
```

Comment:
Questa cosa non è molto chiara generalizza la funziona se viene riutilizzata per più chiamate altrimenti può confondere. Ti consiglio di suddividere in due funzioni specifiche ( una per movie e una per tvshows ) ed in ognuna di loro fare la join con l'altra chiamata direttamente in esse.